.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish package to GitHub Packages
```

Comment:
We can rename it to CD and add all CD jobs in this workflow and then rename the file to CD.yml to stay consistent.

Reply:
okokok